source/faq/faq.rst

```diff
@@ -391,10 +375,8 @@ As the data flow is processed individually for each message, you need to create
 a virtual device to aggregate all attributes, then use this virtual device as
 the input of the flow.
 
-Another thing that you could do is to build a flowbroker node to deal with
-contexts, which can be used to store and retrieve data related to a flow or
-node.
-
+Another thing you can do is use a node that deal with contexts.
+Or you can even create a node that handles contexts.
```

Comment:
Suggestion:
You can also create a node (or use an already existing one) that deals with contexts.